qtpy/__init__.py

```diff
@@ -26,6 +26,13 @@
     >>> from qtpy import QtGui, QtWidgets, QtCore
     >>> print(QtWidgets.QWidget)
 
+PyQt6
+=====
+
+    >>> import os
+    >>> os.environ['QT_API'] = 'pyqt6'
+    >>> from qtpy import QtGui, QtWidgets, QtCore
+    >>> print(QtWidgets.QWidget)
```

Comment:
So seems like this PR also includes some changes to support PyQt6. Checking seems like for some modules there are the additional validations not only for PySide6 but also for PyQt6 (QtCore, QtGui, QtSvg, QtTest, QtWidgets). However, for others is missing (QtDataVisualization, QtHelp, QtNetwork, QtOpenGL, QtPrintSupport, QtQml, QtQuick, QtQuickWidgets, QtSql). As well as in the tests and some other places like the `API_NAME` dictionary and the README. Not totally sure if the scope of this PR is also to do the changes to support PyQt6 but let us know!